src/components/Footer/styles.js

```diff
@@ -0,0 +1,67 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
+  display: flex;
+  justify-content: space-around;
+`;
+
+export const Row = styled.div`
+  min-width: 300px;
+`;
+
+export const Link = styled.a`
+  & > svg {
+    fill: ${(props) => props.theme.hue.gold};
+  }
+`;
+
+export const TextLink = styled.a`
+  color: ${(props) => props.theme.hue.gold};
+  font-size: 20px;
```

Comment:
Remove `font-size`, the default font-size should be sufficient.

Reply:
If "font-size" is removed, the text becomes tiny, is there a better alternative?

Reply:
Hmm, I tested it before it looks fine without explicitly stating `font-size`.

Reply:
![image](https://user-images.githubusercontent.com/90483954/176566012-84b7cb21-5ae5-43bb-9c70-1bf12b807119.png)

Reply:
![image](https://user-images.githubusercontent.com/90483954/176566063-0555df93-416b-47d2-b0d5-3ed55e16cfe5.png)

Reply:
Yup, the font-size looks fine.